common/src/main/java/com/microsoft/identity/common/adal/internal/cache/StorageHelper.java

```diff
@@ -644,34 +729,6 @@ private synchronized KeyPair readKeyPair() throws GeneralSecurityException, IOEx
             // handle it as regular KeyStoreException
             throw new KeyStoreException(e);
```

Comment:
>KeyStoreException [](start = 22, length = 17)

This too in telemetry please.

Reply:
Telemetry is not wired into common yet. We could add this after Heidi has finished wiring her new telemetry work.

Reply:
Best thing I can do here is to bubble up the exception, and have the upper layer (AccountManagerStorageHelper) log the telemetry. Would that work?